From ec9c0a3d90565a5727b66907ccfb77370efb1af2 Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Tue, 26 Jul 2011 08:14:51 -0300
Subject: [RHEL6 qemu-kvm PATCH 1/3] Change snapshot_blkdev hmp to use correct argument type for device

RH-Author: Jes Sorensen <Jes.Sorensen@redhat.com>
Message-id: <1311668092-528-2-git-send-email-Jes.Sorensen@redhat.com>
Patchwork-id: 30386
O-Subject: [PATCH 1/2] Change snapshot_blkdev hmp to use correct argument type for device
Bugzilla: 676982
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Jes Sorensen <Jes.Sorensen@redhat.com>

Pointed out by Markus

Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry-pick with conflicts from 982aa95532a3a7b549695d5b3e18442975eecfb5)
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index c1e9ba7..3c0e23d 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -1212,7 +1212,7 @@ EQMP
 
     {
         .name       = "snapshot_blkdev",
-        .args_type  = "device:s,snapshot_file:s?,format:s?",
+        .args_type  = "device:B,snapshot_file:s?,format:s?",
         .params     = "device [new-image-file] [format]",
         .help       = "initiates a live snapshot\n\t\t\t"
                       "of device. If a new image file is specified, the\n\t\t\t"
-- 
1.7.3.2

